index.html

```diff
@@ -89,6 +90,21 @@
             }
         }
 
+        /* Override bug GitHub button */
+        .mz-bug-container {
+            width: 240px;
+        }
+
+        @media only screen and (max-width: 400px) {
+            .mz-bug-container {
+                width: 144px;
+            }
+        }
+
+        .mz-bug-github-link {
+            display: none;
```

Comment:
What was the context where the github link didn't make sense? (In general I've really liked the ability to link straight back to the source…)